install_config/persistent_storage/persistent_storage_glusterfs.adoc

```diff
@@ -193,7 +181,14 @@ Gluster server administrators using normal POSIX compliant security.
 By default, SELinux does not allow writing from a pod to a remote Gluster
 server.
 
-To enable writing to GlusterFS volumes with SELinux enforcing on each node, run:
+To enable writing to GlusterFS volumes with SELinux enforcing first install
+the`docker-selinux on each node:
+
+----
+# yum install docker-selinux
```

Comment:
@swagiaal Maybe we could just put a NOTE box after the part about `virt_sandbox_use_fusefs` saying that it's provided by the `docker-selinux` package, so that if someone doesn't have it they'd know what to make sure is installed.